ui-react/packages/atlasmap-provider/src/AtlasmapProvider.tsx

```diff
@@ -186,6 +187,7 @@ export function useAtlasmap({
   sources: IAtlasmapDocument[];
   targets: IAtlasmapDocument[];
   mappings: IMappings[];
+  deleteAtlasFile: (fileName: any, isSource: any) => void;
```

Comment:
String and bool?

Reply:
The UI side returns a Mutable string | undefined which doesn't match string.  I didn't want to proliferate that mutable type into the provider side.

Reply:
I’d still explicitly set the types and either wrap the calling functions in an if to check for undefined or use the ! operator to suppress the warning

Reply:
hmm - how does the ! operator work exactly?? :)

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
Added new PR

Reply:
Erh... I was suggesting the other way round, checking that current is not null and making the type required in the delete function

Reply:
Doh!  I'll give it another shot...